src/Resources/app/storefront/src/js/filter-slider-range.plugin.js

```diff
@@ -3,334 +3,437 @@ import DomAccess from 'src/helper/dom-access.helper';
+                this.validateMaxInput();
+                stateChanged = true;
+            }
+        });
+
+        return stateChanged;
+    }
+
+    /**
+     * @param {Array} values
+     */
+    onUpdateValues(values) {
+        if (values[0] < this.options.price.min) {
+            values[0] = this.options.price.min;
+        }
+        if (values[1] > this.options.price.max) {
+            values[1] = this.options.price.max;
+        }
+
+        this._inputMin.value = values[0];
+        this._inputMax.value = values[1];
+    }
+
+    /**
+     * @param {String} id
+     * @public
+     */
+    reset(id) {
+        if (id === this.options.minKey) {
+            this.resetMin();
+        }
+
+        if (id === this.options.maxKey) {
+            this.resetMax();
+        }
+
+        this._removeError();
+    }
+
+    /**
+     * @public
+     */
+    resetAll() {
+        this.resetMin();
+        this.resetMax();
+        this._removeError();
+    }
+
+    validateMinInput() {
+        if (!this._inputMin.value || this._inputMin.value < this.options.price.min || this._inputMin.value > this.options.price.max) {
+            this.resetMin();
+        } else {
+            this.setMinKnobValue();
+        }
+    }
+
+    validateMaxInput() {
+        if (!this._inputMax.value || this._inputMax.value > this.options.price.max || this._inputMax.value < this.options.price.min) {
+            this.resetMax();
+        } else {
+            this.setMaxKnobValue();
+        }
+    }
+
+    resetMin() {
+        this._inputMin.value = this.options.price.min;
+        this.setMinKnobValue();
+    }
+
+    resetMax() {
+        this._inputMax.value = this.options.price.max;
+        this.setMaxKnobValue();
+    }
+
+    /**
+     * @private
+     */
+    _onChangeMin() {
+        this.setMinKnobValue();
+        this._onChangeInput();
+    }
+
+    /**
+     * @private
+     */
+    _onChangeMax() {
+        this.setMaxKnobValue();
+        this._onChangeInput();
+    }
+
+    hasMinValueSet() {
+        this.validateMinInput();
+        return this._inputMin.value.length && parseFloat(this._inputMin.value) !== this.options.price.min;
+    }
+
+    hasMaxValueSet() {
+        this.validateMaxInput();
+        return this._inputMax.value.length && parseFloat(this._inputMax.value) !== this.options.price.max;
+    }
+
+    setMinKnobValue() {
+        if (this.slider) {
+            this.slider.noUiSlider.set([this._inputMin.value, null]);
+        }
+    }
+
+    refreshDisabledState(filter) {
+        const properties = filter[this.options.name];
+        const entities = properties.entities;
+
+        if (!entities) {
+            this.disableFilter();
+
+            return;
+        }
+
+        const property = entities.find(entity => entity.translated.name === this.options.propertyName);
+        const totalRangePrices = property.options[0].totalRange;
+        const alreadySelectedPrices = this.getValues();
```

Comment:
I don' really like that naming, something like `currentSelectedPriced` sound better